setup.py

```diff
@@ -35,5 +35,5 @@
         'serenata_toolbox.datasets'
     ],
     url=REPO_URL,
-    version='12.4.4'
+    version='13.0.0'
```

Comment:
Not related to the version but, but to `setup.py`: If we had to update tests from Python 3.5 to Python 3.6, we might need to change line 13 to remove support for Python 3.5 and explicitly support for Python 3.6.